src/harness/values.yaml

```diff
@@ -525,6 +528,8 @@ platform:
       SSCA: "SSCA_ENABLED,SSCA_MANAGER_ENABLED,SSCA_SLSA_COMPLIANCE"
       # -- STO Feature Flags (activated when global.sto is enabled)
       STO: "STO_BASELINE_REGEX,STO_STEP_PALETTE_BURP_ENTERPRISE,STO_STEP_PALETTE_CODEQL,STO_STEP_PALETTE_FOSSA,STO_STEP_PALETTE_GIT_LEAKS,STO_STEP_PALETTE_SEMGREP"
+      # -- IDP Feature Flags (activated when global.idp is enabled)
+      IDP: "IDP_ENABLED"
```

Comment:
this probably wont work, it requires a cg-manager helm chart change as well. you can have this merge, but lets sync and get the prerequsites done for this.